lisa/node.py

```diff
@@ -1186,6 +1192,9 @@ class NodeHookSpec:
     def get_node_information(self, node: Node) -> Dict[str, str]:
         ...
 
+    @hookspec
+    def check_dmesg_oops(self, node: Node) -> bool:
```

Comment:
eturn the found error logs, so the errors can be in test case error message directly.